packages/tailwind-typography/src/index.ts

```diff
@@ -0,0 +1,205 @@
+import plugin from 'tailwindcss/plugin'
+
+function isPlainObject(x: unknown): x is Record<string, unknown> {
+  return (
+    typeof x === 'object' &&
+    x?.constructor === Object &&
+    Object.getPrototypeOf(x) === Object.prototype
+  )
+}
```

Comment:
**thought:** we should make an internal package of "type predicate" functions

Like this `isPlanObject` function.